package.json

```diff
@@ -12,7 +12,7 @@
     "compile-sass": "sass --quiet-deps --no-source-map --load-path=node_modules/govuk-frontend --load-path=node_modules/@ministryofjustice/frontend --load-path=. assets/scss/application.scss:./assets/stylesheets/application.css assets/scss/application-ie8.scss:./assets/stylesheets/application-ie8.css --style compressed",
     "copy-views": "cp -R server/views dist/server/",
     "int-test-ui": "cypress open",
-    "int-test": "cypress run --config video=false",
+    "int-test": "cypress run",
```

Comment:
Should I be able to see the video in the `test_results/cypress` directory here, or is there any more setup I need to do? I can't seem to find those videos, only the xml reports.

Reply:
You might need to show me what you see

Reply:
as discussed on slack, I was running the wrong script and failing at the unit tests 🤦